kentik_api_library/tests/unit/api_resources/test_alerting.py

```diff
@@ -32,3 +33,119 @@ def test_create_manual_mitigation_success() -> None:
     assert connector.last_payload["minutesBeforeAutoStop"] == "20"
 
     assert created is True
+
+
+def test_get_active_alerts() -> None:
+    # given
+    create_response_payload = """
+    [
+        {
+            "alarm_id": 82867908,
+            "row_type": "Alarm",
+            "alarm_state": "ALARM",
+            "alert_id": 15094,
+            "mitigation_id": null,
+            "threshold_id": 76518,
+            "alert_key": "443",
+            "alert_dimension": "Port_dst",
+            "alert_metric": [
+                "bits"
+            ],
+            "alert_value": 2270.4,
+            "alert_value2nd": 0,
+            "alert_value3rd": 0,
+            "alert_match_count": 5,
+            "alert_baseline": 769,
+            "alert_severity": "minor",
+            "baseline_used": 15,
+            "learning_mode": 0,
+            "debug_mode": 0,
+            "alarm_start": "2021-01-19T13:50:00.000Z",
+            "alarm_end": "0000-00-00 00:00:00",
+            "alarm_last_comment": null,
+            "mit_alert_id": 0,
+            "mit_alert_ip": "",
+            "mit_threshold_id": 0,
+            "mit_method_id": 0,
+            "args": "",
+            "id": 0,
+            "policy_id": 15094,
+            "policy_name": "test_policy1",
+            "alert_key_lookup": "443"
+        }
+    ]"""
+    connector = StubAPIConnector(create_response_payload, HTTPStatus.OK)
+    alerting_api = AlertingAPI(connector)
+
+    # when
+    alarms = alerting_api.get_active_alerts("2020-10-15T22:15:00", "2021-01-20T09:15:00")
```

Comment:
if we return datetime in response objects, maybe we should also accept datetime instead of string?